charts/pega/templates/_helpers.tpl

```diff
@@ -359,10 +359,22 @@ true
     sources:
     - secret:
         name: {{ template "pegaCredentialsSecret" $ }}
-  {{ if and (.Values.global.jdbc.external_secret_name) (not .Values.global.jdbc.password) }}
+  {{ if ((.Values.global.jdbc).external_secret_name) }}
     - secret:
         name: {{ .Values.global.jdbc.external_secret_name }}
   {{- end }}
+  {{ if ((.Values.hazelcast).external_secret_name)}}
+    - secret:
+        name: {{ .Values.hazelcast.external_secret_name }}
```

Comment:
Just to confirm, deployment will not break if the value of Values.hazelcast.external_secret_name, .Values.dds.external_secret_name, Values.global.customArtifactory.authentication.external_secret_name is the same ?

Reply:
Discussed with Giriraj on this, hence resolving

Yes , we can provide same secret name to multiple external_secret_name parameters , this doesn't break deployment